presto-hive/src/main/java/com/facebook/presto/hive/HiveTableProperties.java

```diff
@@ -101,6 +102,17 @@ public HiveTableProperties(TypeManager typeManager, HiveClientConfig config)
                                 .map(name -> ((String) name).toLowerCase(ENGLISH))
                                 .collect(Collectors.toList())),
                         value -> value),
+                new PropertyMetadata<>(
+                        DISTRIBUTION_PROPERTY,
+                        "data distribution given bucket columns",
+                        typeManager.getType(parseTypeSignature("array(varchar)")),
```

Comment:
Is this going to be a list of dim columns? If yes, maybe use name `CLUSTERING_PROPERTY = "clustered_by";`